scripts/validate_schemas.js

```diff
@@ -66,16 +65,24 @@ module.exports = {
 // @ts-ignore
 if (require.main === module) {
   // When this is being called as a script via `node scripts/validate_schemas.js`
-  getBreakingChanges().then(changes => {
-    if (changes.length) {
-      process.exitCode = 1
-      console.error(
-        "Failing due to breaking changes between Force and Metaphysics Production\n\n"
-      )
-      console.error(changes)
-      console.error(
-        "\n\nYou should deploy metaphysics production, and re-deploy force"
-      )
-    }
-  })
+  if (process.argv.length != 3) {
+    console.log('This script must be called with either "staging" or "production"')
+    process.exitCode = 1
+  } else {
+    const env = process.argv[2]
+    getBreakingChanges(env).then(changes => {
+      if (changes.length) {
+        process.exitCode = 1
+        console.error(
+          `Failing due to breaking changes between Force and Metaphysics ${env}\n\n`
```

Comment:
Since the failure details can be a little counterintuitive (e.g., "field removed" when Metaphysics is pending a merge or deploy with a new field), it might help to clarify this header; something like `"Metaphysics' ${env} is incompatible with this project's expectations: ..."`.

Reply:
Done.